templates/uui-nextjs-template-app-router/helpers/apiDefinition.ts

```diff
@@ -0,0 +1,15 @@
+import { IProcessRequest } from "@epam/uui-core";
+
+export function apiDefinition(processRequest: IProcessRequest) {
```

Comment:
Let's remove this helper and usage since using of it isn't required and current implementation does not provide any benefit for user